specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,616 @@
+        }
+      }
+    },
+    "DataPolicy": {
+      "required": [
+        "decisionRules",
+        "members",
+        "description"
+      ],
+      "type": "object",
+      "description": "Purview data policy",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The policy id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The policy name"
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The policy kind"
+        },
+        "source": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The policy source"
+        },
+        "state": {
+          "description": "The state of the policy",
+          "enum": [
+            "Enabled",
+            "Draft"
+          ],
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "state",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "type": "string",
+          "description": "The policy description"
+        },
+        "schemaVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The schema version of the policy"
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The etag version of a policy"
+        },
+        "scopes": {
+          "type": "array",
+          "readOnly": true,
+          "description": "Array of scopes where the policy is published",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
+        "members": {
+          "type": "object",
+          "description": "Members of the policy",
+          "$ref": "#/definitions/PolicyMembers"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/systemData"
+        },
+        "decisionRules": {
+          "type": "array",
+          "description": "Array of decisionRules for the policy",
+          "items": {
+            "$ref": "#/definitions/PolicyDecisionRule"
+          }
+        }
+      }
+    },
+    "PolicyMembers": {
+      "type": "object",
+      "description": "Policy member",
+      "properties": {
+        "fabricItemMembers": {
+          "type": "array",
+          "description": "Array of azure active directory members",
+          "items": {
+            "$ref": "#/definitions/FabricItemMember"
+          },
+          "x-ms-identifiers": []
+        },
+        "aadMembers": {
+          "type": "array",
+          "description": "Array of azure active directory members",
+          "items": {
+            "$ref": "#/definitions/AADMember"
+          },
+          "x-ms-identifiers": []
+        }
```

Comment:
As a user I would scratch my head here: two different properties with different types but the exact same description. I think users will need more information for these to know what they mean / how to set them.

Reply:
As decided during the review, removed the `fabricItemMembers` field.